easybuild/framework/easyconfig/tools.py

```diff
@@ -390,14 +421,24 @@ def parse_easyconfigs(paths, validate=True):
 
     for (path, generated) in paths:
         path = os.path.abspath(path)
+        # Get EasyConfig specific build options if they have been specified in an EasyStack file
+        opts_current_ec = None
+        if opts_per_ec is not None:
+            opts_current_ec = opts_per_ec.get(os.path.basename(path), None)
+        # Apply custom build options if needed
+        if opts_current_ec is not None:
+            _log.debug("Applying easyconfig-specific build options %s to %s" % (opts_current_ec, path))
+            orig_build_options = update_build_options(opts_current_ec)
+            # TODO: we should probably also update the pr_path (?) since process_easyconfig calls EasyConfig(...)
```

Comment:
We should probably update pr_path here as well, in a similar way as we do in `resolve_dependencies` (and we should probably grab that code snipped and make a function out of it, rather than duplicating that code everywhere)